runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -500,17 +512,21 @@ TR_J9ServerVM::classHasBeenExtended(TR_OpaqueClassBlock *clazz)
       // Check if flag is set, since once set it can not be removed
       if((classDepthAndFlags & J9AccClassHasBeenOverridden) != 0)
          {
+         printf("%s(): clazz %p data from the cache: J9AccClassHasBeenOverridden 1\n", __FUNCTION__, clazz);
          return true;
          }
       else
          {
          //Flag not set, check with client as cache data may be outdated
          stream->write(JITServer::MessageType::VM_classHasBeenExtended, clazz);
-         return std::get<0>(stream->read<bool>());
+         bool result = std::get<0>(stream->read<bool>());
```

Comment:
In the new implementation this message will completely go away

Reply:
5d2abab still contains the experiment code. I'll clean it up first.